package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mattrglobal/bbs-signatures",
-  "version": "1.0.0",
```

Comment:
Should this upgrade count as breaking changes?

Reply:
Was checking previously with others, so likely will be considered as breaking changes as we will be dropping support for old node versions.

But will double confirm again with @jhill-mattr @tplooker @colinyip 

Thanks for checking & raising this @kenhuang

Reply:
Could be argued both ways. I have checked with @tplooker and we agree no downside to bumping major version.